airbyte-integrations/connectors/source-gitlab/metadata.yaml

```diff
@@ -23,6 +23,14 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  releases:
+    breakingChanges:
+      2.0.0:
+        message:
+          In this release, pipeline.created_at and pipeline.updated_at fields for stream Deployments
+          and expires_at field for Group Members and Project Members were changed to date-time format as Gitlab API declares.
+          Users will need to refresh the source schema and reset affected streams after upgrading.
```

Comment:
In this release, several streams were updated to date-time format, as declared in the Gitlab API.  These changes impact pipeline.created_at and pipeline.updated_at fields for stream Deployments and expires_at field for stream Group Members and stream Project Members.
Users will need to refresh the source schema and reset affected streams after upgrading.

Reply:
updated